assets/schema/nav-components.yaml

```diff
@@ -0,0 +1,133 @@
+$schema: http://json-schema.org/draft-07/schema#
```

Comment:
Everything below here is just schema to support the data files and test to validate the data files against the schema. The documentation above may be a bit anemic for changing these files, but I think the schema documentation makes up for that.

Hope this doesn't feel like too much bloat. It's... a religious thing....